lib/ansible/module_utils/powershell/Ansible.ModuleUtils.FileUtil.psm1

```diff
@@ -22,7 +23,7 @@ Function Test-AnsiblePath {
     } catch [NotSupportedException] {
         # When testing a path like Cert:\LocalMachine\My, System.IO.File will
         # not work, we just revert back to using Test-Path for this
-        return Test-Path -Path $Path
+        return Test-Path -LiteralPath $Path
```

Comment:
This one is explicitly set as `-Path` as it's meant to be a replacement for `Test-Item` and we use `-Path` as the entry point.